schema/workflow.json

```diff
@@ -76,6 +76,44 @@
         }
       ]
     },
+    "outputDataSchema": {
```

Comment:
This seems a copy of "dataInputSchema"/ "inputDataSchema" as you changed.
Would be better imo to have a single definition called "inputSchema" and then reference it rather than having two separate

Reply:
@tsurdilo I would recommend calling that definition `dataValidation` or whatever, as `inputSchema` would be confusing when used for output 😜

Reply:
maybe inputOutputValidation then cause we are not validating like event payloads or action results or state data which are all data

Reply:
@tsurdilo Point taken! However, I personally dislike that term too. I'm sure we can, however, find a good one. I'll sleep on it.

Reply:
I think we can use "validationSchema" for the data stucture used to validate schema

Reply:
I like that, good idea!